src/main/java/interfaces/Man.java

```diff
@@ -31,9 +32,4 @@ public void clean() {
     protected Object clone() throws CloneNotSupportedException {
```

Comment:
do we need the annotation @Override

Reply:
Why not?

Reply:
@Akselester, we need )

Reply:
@Akselester, we need )

Reply:
This is overriding of Object's method clone(). So we need that annotation.